### PR TITLE
DC-6716 Add deprecation flag and updated README file

### PR DESCRIPTION
**Pull request by Roshangautam2052**
Requested the team plat-ops to add the bobby rule to decommission EmailLibrary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-#emailaddress
+# emailaddress
+### This library is Deprecated.
+### You can reach out Digital-Contact team for further assistance alternatively you can use this service.
+### https://github.com/hmrc/preferences-frontend/blob/main/app/emailaddress/EmailAddress.scala
```
Comment (ardwalker) [inline README.md]: Do you need this line? Is this an example of using an EmailAddress class without using the library? If so, please add a comment as such.

File: build.sbt [hunk 2]
```diff
@@ -21,19 +21,19 @@ lazy val commonSettings =
 lazy val emailaddress = Project(appName, file("."))
   .settings(publish / skip := true)
   .aggregate(
-    play28,
+    //play28,
```
Comment (ardwalker) [inline build.sbt]: I am the worst offender for doing this, but don’t  leave commented code in when you commit. :)
Thanks.

File: build.sbt [hunk 3]
```diff
@@ -21,19 +21,19 @@ lazy val commonSettings =
...
-
-lazy val play28 = Project(s"$appName-play-28", file("play-28"))
-  .settings(
-    crossScalaVersions := Seq(scala2_12, scala2_13),
-    libraryDependencies ++= LibDependencies.compileDependencies(PlayVersion.Play28) +: LibDependencies.testDependencies,
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always,
-    sharedSources
-  )
-  .settings(commonSettings)
+//
```
Comment (ardwalker) [inline build.sbt]: Removing this should be fine, but again, please delete commented out lines

File: project/LibDependencies.scala [hunk 4]
```diff
@@ -15,8 +15,8 @@ object LibDependencies {
   }
 
   def compileDependencies(playVersion: PlayVersion) = playVersion match {
-    case PlayVersion.Play28 => "com.typesafe.play" %% "play" % "2.8.21"
-    case PlayVersion.Play29 => "com.typesafe.play" %% "play" % "2.9.0"
+    //case PlayVersion.Play28 => "com.typesafe.play" %% "play" % "2.8.21"
```
Comment (ardwalker) [inline project/LibDependencies.scala]: Delete

File: project/LibDependencies.scala [hunk 5]
```diff
@@ -15,8 +15,8 @@ object LibDependencies {
...
+    case PlayVersion.Play29 => "com.typesafe.play" %% "play" % "2.9.5"
```
Comment (ardwalker) [inline project/LibDependencies.scala]: Did you need to do this?

File: README.md (hunk 1 above)
Comment (Roshangautam2052) [reply README.md]: Thanks for the comment, I have been suggested by Paul to add a link instead if someone wants to use it. I will add the comment if someone wants to use the class.

File: build.sbt (hunk 2 above)
Comment (Roshangautam2052) [reply build.sbt]: Yeah definitely, I will remove this comment.

File: build.sbt (hunk 3 above)
Comment (Roshangautam2052) [reply build.sbt]: Sure, I will delete these commented lines.

File: project/LibDependencies.scala (hunk 4 above)
Comment (Roshangautam2052) [reply project/LibDependencies.scala]: I will delete this commented lines as well.Thanks.

File: project/LibDependencies.scala (hunk 5 above)
Comment (Roshangautam2052) [reply project/LibDependencies.scala]: I will delete this as well. Thanks.

File: README.md (hunk 1 above)
Comment (ardwalker) [reply README.md]: Yeah, just a comment to that effect, a bit of context is useful when reading this later. Thanks.

File: project/LibDependencies.scala (hunk 5 above)
Comment (ardwalker) [reply project/LibDependencies.scala]: Just the version bump I mean, don’t  delete the line :)

**Review by ardwalker — APPROVED** [review]

**Pull request merged**